### ISSUE TITLE
Is it possible to use projectile-find-file with a prefilled locatedb?

### ISSUE THREAD

**Issue by gizmomogwai** [bbatsov/projectile#1510]
I love projectile, but I work in a very large workspace and would like to used a prefilled locatedb to get to all the files (and even use locate to narrow the search down if possible).
a locate request is quite fast (<1s) for me whereas projectile need quite a while to do the search (10s)

**Comment by bbatsov** [comment]
Yeah, I think that'd be just a matter of adding some check for the existence of the locatedb and leveraging it. PR welcome.

**Comment by gizmomogwai** [comment]
is there a set of functions to implement for this? i am not familiar with projectile's codebase?
